themes/default/content/docs/intro/pulumi-service/team-access-tokens.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Team Access Tokens"
+meta_desc: Overview of Team Access Tokens in the Pulumi Service.
+menu:
+  intro:
+    parent: pulumi-service
+    weight: 2
+---
+
+Team Access Tokens, like [Organization Access Tokens]({{< relref "/docs/intro/pulumi-service/organization-access-tokens" >}}), provide Enterprise and Business Critical customers the opportunity to manage resources and stack operations for their organization independent of a single-user account. However, Team Access Tokens enable this access to be scoped to the stack access of a [Pulumi Team]({{< relref "/docs/intro/pulumi-service/teams" >}}), rather than to the entire organization's stacks.
+
+Collectively Organization Access Tokens and Team Access Tokens are referred to as "machine tokens", which are not owned by a real user in your organization. This distinguishes them from Personal Access Tokens.
+
+Team Access Tokens are available on Enterprise and Business Critical subscriptions, as well as trials.
+
+Team Access Tokens provide several benefits over Organization and Personal Access Tokens:
+
+* They can be managed by Team Admins in addition to Organization Admins, allowing more users in your organization to leverage machine tokens.
+* They support user-independent usage in your CI integrations while having less privileged scope to other stacks in your organization.
+
+## Creating a Team Access Token
+
+Navigate to your organization, then:
+
+1. Select **Teams**.
+1. Select the team you would like to attach the token to.
+1. Scroll to **Access Tokens**.
+
+Team Access Tokens, like Organization Access Tokens, must have a name that is unique among all Organization Access Tokens assigned to it. This allows tokens taking operations on behalf of your organization to be identifiable in the event that one is compromised.
+
+Once you name a token, the name is taken forever, even after you delete it. This is in order to maintain the integrity of Audit Log Events which persist the token's name as part of the event (see below). Any other Organization Admin, or a Team Admin for the associated team, can delete the token since they are managed by the team and not by a user.
+
+The creation of any Team Access Token, and the user who performed it, is logged as an Audit Log Event.
+
+## Viewing Team Access Tokens
+
+To view Team Access Tokens, go to your team's page within your respective organization and scroll to the Team Access Tokens card. Only Organization Admins and Team Admins for this team will see this card.
+
+## Deleting a Team Access Token
+
+A Team Access Token can be deleted by any Admin of this Team or any Organization Admin. No other member types of the organization can delete a Team Access Token.
```

Comment:
```suggestion
A Team Access Token can be deleted by any Team Admin or any Organization Admin. No other member types of the organization can delete a Team Access Token.
```

Reply:
I wanted to be explicit here that only the specific team's admins could great tokens for that team. A bit overkill, but this leaves some ambiguity in that regard. Any way we should marry the two?

Reply:
updated, let me know if that works!